runtime/Cargo.toml

```diff
@@ -11,209 +11,189 @@ repository = "https://github.com/Polkadex-Substrate/Polkadex"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["max-encoded-len"] }
-scale-info = { version = "2.0.1", default-features = false, features = ["derive"] }
-hex-literal = { version = "0.3.3", optional = true }
-static_assertions = "1.1.0"
-log = { version = "0.4.14", default-features = false }
-smallvec = "1.6.1"
+parity-scale-codec = { workspace = true, default-features = false, features = ["max-encoded-len"] }
+scale-info = { workspace = true, default-features = false, features = ["derive"] }
+hex-literal = { version = "0.3.1", optional = true }
```

Comment:
Why this not inherited from the workspace definition?

Reply:
optional dependency without ability to featuregate non-std crate. all workspace dependencies are pre-built, meaning that if it will be included there - it'll try to get built into WASM causing issues... had to stay here.